rust/repo/{{ cookiecutter.repo_name }}/deny.toml

```
@@ -19,6 +19,14 @@ allow = [
     "ISC",
     "MIT",
     "Unicode-DFS-2016",
+{%- if cookiecutter.license == "GPL-3.0-only" or cookiecutter.license == "AGPL-3.0-only" -%}
```

:wrench: a couple of suggestions
1. suggest using the `in` (or `__contains__`) syntax
2. Omit the `-` at the end of the template expression to resultant array as multi-line
```suggestion
{%- if cookiecutter.license in ("GPL-3.0-only", "AGPL-3.0-only") %}
```